From 5286d0fe8115c3b2ea07e74a47a53d45ebb13ed7 Mon Sep 17 00:00:00 2001
From: Felix Hanau <felix@cloudflare.com>
Date: Wed, 26 Jul 2023 18:40:13 +0200
Subject: Fix ICU build

ICU changed how a header is included, but V8's bazel build file has not been updated to make the header available.

Change-Id: Ibe49447319a8aef362f73fb87158e0de80b5d620
---
 bazel/BUILD.icu | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bazel/BUILD.icu b/bazel/BUILD.icu
index 73a25df55abc380092a0177bb13364e6f216ce95..3b26219eb56f4e8247af8dad476ebced2ad763af 100644
--- a/bazel/BUILD.icu
+++ b/bazel/BUILD.icu
@@ -96,7 +96,7 @@ cc_library(
     hdrs = glob([
         "source/common/unicode/*.h",
         "source/i18n/unicode/*.h",
-    ]),
+    ]) + ["source/stubdata/stubdata.h"],
     copts = select({
         "@platforms//os:windows": [
             "/wd4005",  # Macro redefinition.
